### PR TITLE
Prepare for 6.0.1 release.

### PR DESCRIPTION
**Pull request by usergenic**
* Updated package.json dependencies:
  * Upgraded @types/gulp
  * Moved all @types to devDependencies
  * Removed PolymerElements/test-fixture from npm dependencies (is already installed by bower)
* [x] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by usergenic** [comment]
This release exists primarily to address https://github.com/Polymer/polymer-cli/issues/754

**Comment by rictic** [comment]
Hm, it looks like it doesn't compile with the latest typescript.

**Comment by rictic** [comment]
Mind fixing?

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "@types/minimatch": "^2.0.29",
     "@types/mocha": "^2.2.32",
     "@types/multer": "0.0.32",
-    "@types/node": "4.0.30",
+    "@types/node": "^7.0.27",
```
Comment (stramel) [inline package.json]: Latest is now ^8.0.1

**Comment by web-padawan** [comment]
@usergenic FYI there used to be a [milestone](https://github.com/Polymer/web-component-tester/milestone/3), and what about taking a look at #578?

**Comment by usergenic** [comment]
@web-padawan i'll be looking at the other WCT PRs shortly.  Had an immediate fix I had to make to WCT for compilation errors and to update dependencies in preparation for CLI update which includes WCT and other things.  Thanks for pointing out the milestone.  I'll update.

**Pull request merged**